index-legacy.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Delta UI</title>
+    <link rel="stylesheet" href="/styles.css">
+</head>
+<body>
+    <div class="container">
+        <section class="container__header">
+            <!-- Header Component Starts -- Standard -->
+            <div class="header header--standard">
+                <button class="header__btn button-inherit-parent">
+                    <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-3d-cube-sphere" width="56" height="56" viewBox="0 0 24 24" stroke-width="1.5" stroke="#ffffff" fill="none" stroke-linecap="round" stroke-linejoin="round">
+                        <path stroke="none" d="M0 0h24v24H0z" fill="none"/>
+                        <path d="M6 17.6l-2 -1.1v-2.5" />
+                        <path d="M4 10v-2.5l2 -1.1" />
+                        <path d="M10 4.1l2 -1.1l2 1.1" />
+                        <path d="M18 6.4l2 1.1v2.5" />
+                        <path d="M20 14v2.5l-2 1.12" />
+                        <path d="M14 19.9l-2 1.1l-2 -1.1" />
+                        <line x1="12" y1="12" x2="14" y2="10.9" />
+                        <line x1="18" y1="8.6" x2="20" y2="7.5" />
+                        <line x1="12" y1="12" x2="12" y2="14.5" />
+                        <line x1="12" y1="18.5" x2="12" y2="21" />
+                        <path d="M12 12l-2 -1.12" />
+                        <line x1="6" y1="8.6" x2="4" y2="7.5" />
+                      </svg>
+                </button>
+                <h3 class="header__title">
+                    Delta UI
+                </h3>
+
+                <div class="header__action">
+                    <!-- <button class="header__btn button-inherit-parent">
+                        <img class="header__icon" src="/components/header/download.svg" alt="Image">
+                    </button>
+                    <button class="header__btn button-inherit-parent">
+                        <img class="header__icon" src="/components/header/printer.svg" alt="Image">
+                    </button>
+                    <button class="header__btn button-inherit-parent">
+                        <img class="header__icon" src="/components/header/bookmark.svg" alt="Image">
+                    </button> -->
```

Comment:
remove commented code.

Reply:
Part of legacy file. will be deleted shortly.